torch/distributed/pipelining/schedules.py

```diff
@@ -1838,43 +1845,49 @@ def __init__(
         args_chunk_spec: Optional[Tuple[TensorChunkSpec, ...]] = None,
         kwargs_chunk_spec: Optional[Dict[str, TensorChunkSpec]] = None,
         output_merge_spec: Optional[Union[Dict[str, Any], Tuple[Any]]] = None,
+        enable_zero_bubble: bool = False,
```

Comment:
hm, why is there an 'enable_zero_bubble' option for 1F1B?  How is 1F1B+ZB=True different from the ZB1P schedule defined in the other class?

Reply:
good catch, I accidentally left this in. It should be removed